src/Fields.php

```diff
@@ -1070,6 +1120,12 @@ protected function wf_crm_get_fields($var = 'fields') {
           $fields[$id]['extra']['cols'] = $custom_field['note_columns'] ?? 60;
           $fields[$id]['extra']['rows'] = $custom_field['note_rows'] ?? 4;
         }
+        // Set maximum field length.
+        if (isset($custom_field['text_length'])) {
```

Comment:
Thanks @MegaphoneJon I've tested this and it works as expected. One minor correction though -

It seems `counter_type`, `counter_maximum`, etc is a property specific to textfields and textarea but with this PR, it seems it adds the counter_type to all the custom fields irrespective of its type? Can we add an additional check for type = text at this point?

`text_length` in civi is a Database field length and is set for checkbox, radio custom fields as well. Setting counter_type for these elements on the webform doesn't fix anything. So seems like adding unnecessary lines to the yaml source.